src/main/java/org/mockito/junit/MockitoJUnit.java

```diff
@@ -1,29 +1,32 @@
-package org.mockito.junit;
-
-/**
- * The JUnit rule can be used instead of {@link org.mockito.runners.MockitoJUnitRunner}. See {@link MockitoRule}.
- *
- * @since 1.10.17
- */
-public class MockitoJUnit {
-
-    /**
-     * Creates rule instance that initiates &#064;Mocks
-     * See {@link MockitoRule}.
-     *
-     * @return the rule instance
-     */
-    public static MockitoRule rule() {
-        return new MockitoJUnitRule();
-    }
-
-    /**
-     * Creates a rule instance that can perform lazy verifications.
-     *
-     * @see VerificationCollector
-     * @return the rule instance
-     */
-    public static VerificationCollector collector() {
-        return new VerificationCollectorImpl();
-    }
-}
+package org.mockito.junit;

```

Comment:
It seems like eclipse reformatted the whole file (Unix vs Win line ending). I guess it is related to [this eclipse bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=361503). I will test the workaround described in [this stackoverflow answer](http://stackoverflow.com/a/18696063/932656).